kernel/src/main/java/org/kframework/krun/KRunOptions.java

```diff
@@ -229,14 +225,7 @@ public SearchType searchType() {
         public boolean statistics = false;
 
         @Parameter(names="--debugger", description="Run an execution in debug mode.")
-        private boolean debugger = false;
-
-        public boolean debugger() {
-            if (debugger && search()) {
-                throw new ParameterException("Cannot specify --search with --debug. In order to search inside the debugger, use the step-all command.");
-            }
-            return debugger;
-        }
+        public boolean debugger = false;
```

Comment:
Are we intending to not support Java backend debugger anymore? Fine if so, but let's remove that code then.

If we are intending to, then maybe we should leave this guard in?